packages/libnvidia-container/Cargo.toml

```
@@ -0,0 +1,23 @@
+[package]
+name = "libnvidia-container"
+version = "0.1.0"
+edition = "2018"
+publish = false
+build = "build.rs"
+
+[lib]
+path = "pkg.rs"
+
```

```suggestion

[package.metadata.build-package]
releases-url = "https://github.com/NVIDIA/libnvidia-container/releases"

```

We started including the `releases-url` with packages.